### ISSUE TITLE
Many internal line breaks do not trigger E303

### ISSUE THREAD

**Issue by asottile** [PyCQA/pycodestyle#892]
For example:

```python
x = (
    1,







    2,
)
```

I expect this to trigger E303, however it does not

**Comment by asottile** [comment]
Duplicate of #737
